docker-compose.yml

```diff
@@ -1,16 +0,0 @@
-version: "3.2"
```

Comment:
My only question for this PR is why we no longer need the docker-compose file

Reply:
It hasn't been used since @marcellp migrated the repo away from `remoter`.  Just a bit of housekeeping

Reply:
Just in case I tried restoring this but it did not resolve the issue we are seeing in staging

Reply:
I'll take a look today and try to fix it.